core/src/main/java/io/github/belgif/rest/guide/validator/output/JsonOutputProcessor.java

```diff
@@ -0,0 +1,77 @@
+package io.github.belgif.rest.guide.validator.output;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import io.github.belgif.rest.guide.validator.core.OpenApiViolationAggregator;
+import io.github.belgif.rest.guide.validator.core.Violation;
+import io.github.belgif.rest.guide.validator.output.model.ViolationFileObject;
+import io.github.belgif.rest.guide.validator.output.model.ViolationGroup;
+import io.github.belgif.rest.guide.validator.output.model.ViolationReport;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class JsonOutputProcessor extends OutputProcessor implements DirectoryOutputProcessor {
+    private static final String FILE_NAME = "validationReport.json";
```

Comment:
Because it's only a single output file here, it makes sense that the user can specify the entire output file location (path + file name) from a single property. Especially useful when running from CLI.
You could use ${project.build.directory}/validationReport.json as default, like [checkstyle plugin does](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#outputFile)